### PR TITLE
[New Rule] PowerShell Suspicious Script with Screenshot Capabilities

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Detects PowerShell Scripts that can take screenshots, which is a common feature in post-exploitation kits and RATs.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/collection_posh_screen_grabber.toml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+[metadata]
+creation_date = "2021/10/19"
+maturity = "production"
+updated_date = "2021/10/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects PowerShell Scripts that can take screenshots, which is a common feature in post-exploitation kits and RATs.
```
Comment (brokensound77) [inline rules/windows/collection_posh_screen_grabber.toml:9]:
```suggestion
Detects PowerShell Scripts that can take screenshots, which is a common feature in post-exploitation kits and RATs (Remote Access Tools).
```

will need to re-lint for length

File: rules/windows/collection_posh_screen_grabber.toml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+description = """
+Detects PowerShell Scripts that can take screenshots, which is a common feature in post-exploitation kits and RATs
+(Remote Access Tools).
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "PowerShell Suspicious Script with Screenshot Capabilities"
+risk_score = 47
```
Comment (brokensound77) [inline rules/windows/collection_posh_screen_grabber.toml:17]: no references?

**Review by brokensound77 — APPROVED** [review]
🚀

File: rules/windows/collection_posh_screen_grabber.toml (hunk 2 above)
Comment (w0rk3r) [reply rules/windows/collection_posh_screen_grabber.toml:17]: Added a reference to the api call

**Review by Samirbous — APPROVED** [review]
👍

**Pull request merged**